src/ms.tex

```diff
@@ -284,7 +284,49 @@ \subsection{Connections with Observatories and Missions}
 \secunfilled
 
 Cherenkov Telescope Array
-\secauthor{Axel Donath, Max Noethe}
+\secauthor{Axel Donath, Maximilian Nöthe}
+
+The Cherenkov Telescope (CTA) will be the next generation very-high-energy
+gamma-ray observatory.
+It will improve over the current generation of imaging atmospheric Cherenkov telescopes (IACTs)
+by a factor of five to ten in sensitivity and it will be able to observe the whole sky from a combination of two sites:
+a northern site in La Palma, Spain, and a southern one in Paranal, Chile.
+CTA will be able to observe gamma rays in the energy range between $20\,\mathrm{GeV}$ and $300\,\mathrm{GeV}$
+using three different types of telescopes, in total over 100 telescopes are planned at the two sites.
+CTA will also be the first open gamma-ray observatory.
+
+The data analysis pipeline is developed as open source software and essentially split in two domains:
+\begin{enumerate}
+  \item In the low-level analysis, the properties of the recorded air shower events
+    have to be estimated from the raw data.
+    The raw data consists of very short (\textasciitilde $40\,\mathrm{ns}$) videos recorded with the fast and
+    sensitive cameras of the telescopes.
+    This includes the energy, particle type and direction of origin of the particle that induced the air shower
+    and the time the shower was recorded.
+  \item In the higher-level analysis, these reconstructed event lists are used together with the
+    characterization of the measurement process and the low-level analysis to perform the actual scientific analysis.
+    This includes energy spectra, light curves, sky maps and many more.
+\end{enumerate}
+
+A prototype for the low-level analysis is ctapipe~\cite{ctapipe},
+a python package developed to perform all the necessary tasks to from the raw data
+of Cherenkov telescopes to the reconstructed event lists.
+
+The high-level analysis will be based on the Gammapy~\cite{gammapy} library, which
```

Comment:
Would mention, that `Gammapy` is an Astropy affiliated package here, also the information seem partly duplicated, as one could understand "high-level analysis" and "science tools" as synonymous.

Maybe shorter:
```
The high level analysis (or CTA science tools) will be based on the Astropy affiliated package
Gammapy~\cite{gammapy}.
```